### ISSUE TITLE
workbench error on fresh install

### ISSUE THREAD

**Issue by sutram** [VSpaceCode/VSpaceCode#73]
Hi,

I think it is neat that you guys are bringing Spacemacs keybindings to VSCode. It helps me try out VSCode.

I am running VSpaceCode 0.6 with VCodeVim version 1.15.0 on a fresh VSCode install on Ubuntu 20.04.

I opted for automatic configuration but I am running into the following error for the command "Space f f"

TaskQueue: Error running task. Failed to handle key= . command 'workbench.action.files.openFileFolder' not found.

Thanks!

**Comment by MarcoIeni** [comment]
I get the same error. Maybe this action was removed.

Should we replace it [here](https://github.com/VSpaceCode/VSpaceCode/blob/a0e52c766b20157dfe33b335e69b3b45d4d5aa0b/package.json#L372) with `workbench.action.files.openFile`?

Add this to your `settings.json` as a temporary workaround:
```json
{
  "vspacecode.bindingOverrides": [
    {
      "keys": "f.f",
      "name": "Open file",
      "type": "command",
      "command":"workbench.action.files.openFile",
    }
  ]
}
```

**Comment by stevenguh** [comment]
Apparently, `workbench.action.files.openFileFolder` only works on mac, and `workbench.action.files.openFile` doesn't work on mac. This is such a weird design. I am thinking instead of fixing one while introducing bug for another system. I am thinking to bundle `vscode-file-browser` for `<spc> f f` to fix this so we have same command for different systems (#65). However, vs remote support in `vscode-file-browser` is still being ironed out.

**Comment by sutram** [comment]
@MarcoIeni  Thanks for the workaround. That worked.

@stevenguh I appreciate the work on this extension and the challenge across multiple OSes.

In emacs, it seems to be much easier to set something up like this using (eq system-type 'gnu/linux) :)

**Comment by stevenguh** [comment]
Released `v0.7.0` to use `bodil.file-browser` extension for `<spc> f f`. This will make that command agnostic to platform; although there are some known issues with `bodil.file-browser` in VSCode Remote. I think the author is on it and I think it is small enough to bundle it to fix this issue. The known issues are listed in README:

- File browser (bound to f f) doesn't work correctly when the active document is a local file with VScode Remote
- File browser (bound to f f) doesn't have a button to open local file with VSCode Remote
---
@sutram If you update VSpaceCode to `v0.7.0`, you should be able to remove the overrides in the settings for this bug.
